### ISSUE TITLE
LG: Tap on HeroCard/ThumbnailCard removed when evaluating structured response template

### ISSUE THREAD

**Issue by brett-1hw** [microsoft/botbuilder-dotnet#2958]
## Version
4.6.2-preview

## Describe the bug
Create an LG structured response template with a HeroCard or Thumbnail card:

```
# HeroCard
[HeroCard
   Tap = @{HeroCard.Action()}
   Title = Card Title
]

# HeroCard.Action
[CardAction
    Type = imBack
    Value = Tap Value
]

```
When the template is rendered as an activity, the 'Tap' action will be missing.

## Expected behavior
Tap action is present with all other card properties

[bug]

**Comment by axelsrz** [comment]
@vishwacsena could you take a look at this?

**Comment by Danieladu** [comment]
Currently we just support for limited properities, but soon we will support almost all properties.

**Comment by Danieladu** [comment]
fix by #2996
